src/cpptools.ts

```diff
@@ -491,49 +486,54 @@ export class CppConfigurationProvider implements cpt.CustomConfigurationProvider
      * relative paths
-     * @param grp The file group
-     * @param opts Index update options
+     * @param fileGroup The file group
+     * @param options Index update options
      */
-    private _updateFileGroup(sourceDir: string,
-        grp: codemodel_api.CodeModelFileGroup,
-        opts: codemodel_api.CodeModelParams,
+    private updateFileGroup(sourceDir: string,
+        fileGroup: codeModelApi.CodeModelFileGroup,
+        options: codeModelApi.CodeModelParams,
         target: TargetDefaults,
         sysroot: string) {
-        const configuration = this._buildConfigurationData(grp, opts, target, sysroot);
-        for (const src of grp.sources) {
-            const abs = path.isAbsolute(src) ? src : path.join(sourceDir, src);
-            const abs_norm = util.platformNormalizePath(abs);
-            if (this._fileIndex.has(abs_norm)) {
-                this._fileIndex.get(abs_norm)!.set(target.name, {
-                    uri: vscode.Uri.file(abs).toString(),
+        const configuration = this.buildConfigurationData(fileGroup, options, target, sysroot);
+        for (const src of fileGroup.sources) {
+            const absolutePath = path.isAbsolute(src) ? src : path.join(sourceDir, src);
+            const normalizedAbsolutePath = util.platformNormalizePath(absolutePath);
+            if (this.fileIndex.has(normalizedAbsolutePath)) {
+                this.fileIndex.get(normalizedAbsolutePath)!.set(target.name, {
+                    uri: vscode.Uri.file(absolutePath).toString(),
                     configuration
                 });
             } else {
-                const data = new Map<string, cpt.SourceFileConfigurationItem>();
+                const data = new Map<string, cppTools.SourceFileConfigurationItem>();
                 data.set(target.name, {
-                    uri: vscode.Uri.file(abs).toString(),
+                    uri: vscode.Uri.file(absolutePath).toString(),
                     configuration
                 });
-                this._fileIndex.set(abs_norm, data);
+                this.fileIndex.set(normalizedAbsolutePath, data);
             }
-            const dir = path.dirname(abs_norm);
-            if (this._workspaceBrowseConfiguration.browsePath.indexOf(dir) < 0) {
-                this._workspaceBrowseConfiguration.browsePath.push(dir);
+            const dir = path.dirname(normalizedAbsolutePath);
+            if (this.workspaceBrowseConfiguration.browsePath.indexOf(dir) < 0) {
+                this.workspaceBrowseConfiguration.browsePath.push(dir);
             }
         }
     }
 
     /**
-   * Gets the version of Cpptools API.
-   */
-    get cpptoolsVersion(): cpt.Version {
+     * Version of Cpptools API
+     */
+    private _cpptoolsVersion: cppTools.Version = cppTools.Version.latest;
+
+    /**
+     * Gets the version of Cpptools API.
+     */
+    get cpptoolsVersion(): cppTools.Version {
```

Comment:
get/set doesn't do anything fancy, so we can remove it and make `_cpptoolsVersion` public (renamed to `cpptoolsVersion`)